coffeecard/CoffeeCard.Models/Entities/Product.cs

```diff
@@ -24,6 +24,11 @@ public sealed class Product : IEquatable<Product>
 
         public ICollection<ProductUserGroup> ProductUserGroup { get; set; }
 
+        /// <summary>
+        /// The menu item(s) that this product is eligible to redeem
+        /// </summary>
+        public ICollection<MenuItem> MenuItems { get; set; }
```

Comment:
Suggest to call it EligibleMenuItems or similar to clearly express the relationship

Reply:
Addressed in fe39ffed2d7d416aee95f353586cf80e70afeb6f